presto-tests/src/test/java/com/facebook/presto/tests/TestDictionaryAggregation.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.tests;
+
+import com.facebook.presto.testing.LocalQueryRunner;
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.SystemSessionProperties.DICTIONARY_AGGREGATION;
+import static com.facebook.presto.testing.TestingSession.testSessionBuilder;
+
+public class TestDictionaryAggregation
+        extends AbstractTestQueryFramework
+{
+    public TestDictionaryAggregation()
+    {
+        super(() -> new LocalQueryRunner(testSessionBuilder()
+                .setSystemProperty(DICTIONARY_AGGREGATION, "true")
+                .build()));
+    }
+
+    @Test
+    public void testMixedDistinctWithFilter()
+    {
+        assertQuery(
+                "SELECT " +
+                        "     count(DISTINCT x) FILTER (WHERE x > 0), " +
```

Comment:
this indentation is odd

Reply:
it was copied..

Reply:
i mean: this is formatted according to rules.